Gameplay.java

```diff
@@ -0,0 +1,226 @@
+        	g.setFont(new Font("serif",Font.PLAIN,20));
+        	g.drawString("PRESS LEFT OR RIGHT TO START",190,350);
+        	
+        	//game name
+        	g.setColor(Color.white);
+        	g.setFont(new Font("serif",Font.PLAIN,40));
+        	g.drawString("BREAKOUT BALL",180,240);
+        	
+        	//hide ball
+            g.setColor(Color.black);
+            g.fillRect(110,340,40,40);
+            
+            //hide bricks
+            g.setColor(Color.black);
+            g.fillRect(80,40,540,150);
+    	}
+    	    	
+    	if(totalBricks ==0) {
+    		play = false;
+    		ballXdir = 0;
+    		ballYdir = 0;
+    		g.setColor(Color.green);
+    		g.setFont(new Font("serif", Font.BOLD,30));
+        	g.drawString("You Won !!! Score:"+score, 190,300);
+        	
+        	g.setFont(new Font("serif", Font.BOLD,20));
+        	g.drawString("Press Enter to Restart", 230,350);
+    		
+    	}
+    	
+    	if(ballposY >570) {
+    		play = false;
+    		ballXdir = 0;
+    		ballYdir = 0;
+    		g.setColor(Color.red);
+    		g.setFont(new Font("serif", Font.BOLD,30));
+        	g.drawString("Game Over :D Score:"+score, 190,300);
+        	
+        	g.setFont(new Font("serif", Font.BOLD,20));
+        	g.drawString("Press Enter to Restart", 230,350);
+    	}
+    	
+    	g.dispose();
+    	
+    	
+    }
+	@Override
+	public void actionPerformed(ActionEvent e) {
+		timer.start();
+		if(play) {
+			if(new Rectangle(ballposX,ballposY,20,20).intersects(new Rectangle(playerX,550,100,5))) {
+				ballYdir = -ballYdir;
+			}
+		  A:for(int i =0;i<map.map.length;i++)
+			{
+				for(int j =0;j<map.map[0].length;j++) {
+					if(map.map[i][j]>0) {
+						int brickX = j*map.brickWidth + 80;
+						int brickY = i*map.brickHeight +50;
+						int brickWidth = map.brickWidth;
+						int brickHeight = map.brickHeight;
+						
+						Rectangle rect = new Rectangle(brickX,brickY,brickWidth,brickHeight);
+						Rectangle ballRect = new Rectangle(ballposX,ballposY,20,20);
+						Rectangle brickRect =  rect;
+						
+						if(ballRect.intersects(brickRect)) {
+							map.setBrickValue(0,i,j);
+							totalBricks--;
+							score +=5;
+							
+							if(ballposX +19<=brickRect.x || ballposX + 1>=brickRect.x + brickRect.width){
+								ballXdir = -ballXdir;
+								}else {
+									ballYdir = -ballYdir;
+								}
+						        break A;	
+						}
+						
+					}
+				}
+			}
+
+			ballposX +=ballXdir;
+			ballposY += ballYdir;
+		 if(ballposX <0) {
+			 ballXdir = -ballXdir;
+		 }
+		 if(ballposY <0) {
+			 ballYdir = -ballYdir;
+		 }
+		 if(ballposX >670) {
+			 ballXdir = -ballXdir;
+		 }
+		}
+		
+		repaint();
+		
+	}
+
+	@Override
+	public void keyTyped(KeyEvent e) {}
+	
+	@Override
+	public void keyReleased(KeyEvent e) {}
+
+    @Override
+	public void keyPressed(KeyEvent e) {
+    	if(e.getKeyCode()== KeyEvent.VK_RIGHT) {
+    		if(playerX >570) {
+    			playerX = 580;
+    		}
+    		else
+    		{
+    			moveRight();
+    	    }
```

Comment:
Some parts of your code follow:

```
else {
   ...
}
```

and some follow:

```
else
{
...
}
```

Please stick to one style or let your formatter handle it :D

Reply:
👍